### ISSUE TITLE
disableScroll doesn't work on latest iOS on iPhone 6

### ISSUE THREAD

**Issue by batuhan** [driftyco/ionic-plugin-keyboard#90]
Hi,

I've tried this plugin a while ago and it was working perfectly, I tried it today and somehow the screen bounces when the keyboard is active. 

Code I'm using to activate the feature (this runs inside $ionicPlatform.ready()):
cordova.plugins.Keyboard.disableScroll(true);

This is what happens when an input is focused: http://vitrin.io/static/ionic-keyboard-bug-demo.html

Am I doing something wrong?

Thanks.

**Comment by batuhan** [comment]
It's working correctly now. Wonder what was the issue.

**Comment by batuhan** [comment]
Started experiencing the same issue. Is this related to the plugin or something else?

**Comment by Grigorius** [comment]
Are you using wkwebview ?

**Comment by batuhan** [comment]
This issue appeared before that too but yes. That could be the reason why it returned. 









—


Batuhan İçöz
Lead Developer


E: batuhan@nod.com.trG: +90 (538) 878 10 16T: +90 (850) 885 08 41 (Dahili: 1000)A: Şahkulu Mah. Serdar-ı Ekrem Sok. No: 15/A Galata / İstanbul

**Comment by bFlood** [comment]
I see the same thing. Using AppGyver but I don't think it uses WKWebView

**Comment by batuhan** [comment]
We don't use WKWebView anymore and still have this issue.

**Comment by eroh92** [comment]
I am having the same issue in the iPhone 6 simulator, iOS 8.4. No matter how many times I call disable scroll, it bounces. Randomly, it will work.

**Comment by eroh92** [comment]
Adding this seems to resolve it:
```
  $scope.$on('$cordovaKeyboard:show', function(){
    $cordovaKeyboard.disableScroll(true);
  });
```
Note: I'm using `$cordovaKeyboard` from [ng-cordova](http://ngcordova.com/).

**Comment by eroh92** [comment]
Unfortunately, the issue has returned and I am not using WKWebView. I have disableScroll all over the codebase and nothing seems to be helping? @batuhan @bFlood were you able to get to the bottom of this?

**Comment by bFlood** [comment]
no, not at all. and the keyboard randomly crashes the app as well. I wish appgyver would spend some time on something as important as the keyboard

**Comment by jason-engage** [comment]
@tlancina Tim, if I send you an invite to a private bitbucket repo, can you build it and verify the form bug on IOS? If so, plz send me an email address that I can send you the info and I trust ya'll will keep my code private. Cheers

**Comment by tlancina** [comment]
@jason-engage that should work fine, bitbucket is free to sign up right? Email is tim at ionic.io, thanks!

**Comment by jason-engage** [comment]
Yeah its free. You'll have the invite to join my repo by tomorrow morning.

**Comment by gregavola** [comment]
Has anyone fixed this issue yet?

**Comment by eroh92** [comment]
@tlancina @jason-engage, have you gotten to the bottom of this? Happy to help reproduce if not.

**Comment by jason-engage** [comment]
Nope, I haven't solved it yet.

On Tue, Oct 13, 2015 at 12:55 AM, Rob Eroh <notifications@github.com> wrote:

> @tlancina <https://github.com/tlancina> @jason-engage
> <https://github.com/jason-engage>, have you gotten to the bottom of this?
> Happy to help reproduce if not.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/driftyco/ionic-plugin-keyboard/issues/90#issuecomment-147601136>
> .
>



-- 


[image: ENGAGE] <http://www.engageify.com/>
*        Jason Shaw <jason@engageify.com>*, Director of Operations    Tel.
647 783 7463 | Skype:  jason.engage    en.gg |  <http://en.gg/>Follow us on
Facebook <http://www.facebook.com/engagetheweb>

**Comment by niyando** [comment]
I can confirm this. disableScroll isn't working on iphone 6. It does work for older version (5, 5s)

**Comment by patrickhofer** [comment]
Yes, for me as well

**Comment by patrickhofer** [comment]
I solved this issue by adding `<script>window.cordova = {};</script>` before every scipt in index.html. Seems like window.cordova isn't defined... It's a hack, but it works 😄

**Comment by jetma** [comment]
```
window.addEventListener('native.keyboardshow', function(event) {
	// IMPORTANT!! 
	$timeout(function() {
		$cordovaKeyboard.disableScroll(true);

		// find your own elements and set height to 100%
		var el = angular.element('.modal-backdrop.active ion-content');
		if (el) {
			el.css('height', '100%');
		}
	}, 100);
});
```
